### PR TITLE
Lint the hero of time

### PR DESCRIPTION
**Pull request by BenHSmith13**
Adds lint and travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - nvm install 6.9.1
+  - nvm use 6.9.1
+  - npm install -g yarn
+
+install:
+  - yarn
+
+before_script:
+  - export CHROME_BIN=chromium-browser
+#  - "export DISPLAY=:99.0"
```
Comment (bfcoder) [inline .travis.yml]: These are usually required to get karma specs running

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+#  - "sh -e /etc/init.d/xvfb start"
+
+script:
+  - ./bin/ci
```
Comment (bfcoder) [inline .travis.yml]: This file runs rspec, is that needed here? If so add that file to this PR.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  - ./bin/ci_karma
```
Comment (bfcoder) [inline .travis.yml]: This runs the karma specs, obviously, and that file also needs to be added to this PR.

File: .travis.yml (hunk 2 above)
Comment (BenHSmith13) [inline .travis.yml]: I don't know this isn't done yet. Got busy last night

File: .travis.yml (hunk 3 above)
Comment (BenHSmith13) [inline .travis.yml]: well yeah, I hadn't made it that far yet, James ಠ_ಠ

**Comment by BenHSmith13** [comment]
Ok, @bfcoder it's finished now. Go ahead and review it at your own discosure

**Comment by bfcoder** [comment]
Add the travis ci build passing to the readme too. Other than that this looks good!

**Comment by bfcoder** [comment]
Looks good!

**Pull request merged**